packages/svelte/src/lib/overlay/styles/command.css

```
@@ -0,0 +1,125 @@
+@reference '$style/index.css';
+
+@layer components {
+	[data-command-dialog] {
+		top: 25dvh;
+		left: calc(50lvw - --spacing(4));
+		transform: translate(-50%, 0);
+		max-height: calc(75dvh - --spacing(8));
+
+		[data-command-input] {
+			@apply border-onBackground/20 border border-solid;
+			background-color: var(--color-background);
```

Stylelint (17.7.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)